### ISSUE TITLE
[Quest] The Hunter and the Prince 

### ISSUE THREAD

**Issue by Abusee** [WoWManiaUK/Redemption#1586]
quest the hunter and the prince in icc after turning into arthas has ability to barry and restore 25% of mana the ability never restore mana and i oom and cant defeat illiden 
thanx

**Comment by fvicaria** [comment]
Please follow the procedure described below before creating a new issue:
[How to Properly Report an Issue](https://github.com/wow-mania/Redemption/wiki/How-to-Properly-Report-an-Issue)

This will help developers identify and fix the issue quicker. Incorrectly or unclear reports will simply be deleted.

**Comment by Abusee** [comment]
i tried it agian but still the first ability parry isnot restore any mana and arthas isnot showing even auto attack
thanx

**Comment by captainloozer** [comment]
I have tested this on dev.  Strangely enough, I tried twice, both times there were two Illidans that spawned.  Regardless, I was able to kill one of them and get the second one to half health.  I received credit for completing the quest after killing the first one and I still had half my mana left over.  The quest is completely doable with this amount as shown in the video.   Just make sure you're knocking him back with the number 3 ability and using the number 4 ability to do massive damage to him.  

https://youtu.be/06qmiXJ-MSU

Since I'm sure having to fight 2 Illidans at once is not intended, I will confirm this issue "fixed in dev" since it is doable and allow Janitor to close it if he is pleased with the test results.
